spec/jobs/update_signature_counts_job_spec.rb

```diff
@@ -331,6 +331,68 @@
           expect(Appsignal).to have_received(:send_exception).with(an_instance_of(RuntimeError))
         end
       end
+
+      context "when the count has never been updated before" do
+        let(:petition) { FactoryBot.create(:pending_petition) }
+
+        before do
+          Site.signature_count_updated_at!(nil)
+        end
+
+        context "and there are no validated signatures" do
+          it "updates Site#signature_count_updated_at" do
+            expect {
+              described_class.perform_now(current_time.iso8601)
+            }.to change {
+              Site.signature_count_updated_at
+            }.from(nil).to(current_time - 30.seconds)
```

Comment:
wait `30.seconds`?

Reply:
is this the update interval? maybe it's better to say `interval.seconds`?

Reply:
The signature counting window runs one window behind to ensure that all pending transactions have been committed since the validated_at timestamp is generated in Ruby not the database. If we didn’t then a pending signature validation could be missed.

The reason for not doing `interval.seconds` is it’s tie the expected value to the setup and if for some reason it was changed to something else accidentally the test wouldn’t fail. I alway prefer literal values for expectations wherever possible.

Reply:
This is the historical counting PR: https://github.com/alphagov/e-petitions/pull/712

Reply:
> The reason for not doing interval.seconds is it’s tie the expected value to the setup and if for some reason it was changed to something else accidentally the test wouldn’t fail. I alway prefer literal values for expectations wherever possible.

but even though it was defined on the spec itself?
```ruby
RSpec.describe UpdateSignatureCountsJob, type: :job do
  let(:current_time) { Time.current.change(usec: 0) }
  let(:interval) { 30 }
```
indeed it could be changed accidentally, but so could other `let()` definitions, no?